content/en/blog/2025/otel-arrow-phase-2.md

```diff
@@ -0,0 +1,66 @@
+---
+title: {{ replaceRE "[-_]" " " .Name | title }}
+linkTitle: OpenTelemetry Protocol with Apache Arrow Phase 2
+date: {{ dateFormat "2006-01-02" .Date }}
+author: >-
+  [Joshua MacDonald](https://github.com/jmacd) (Microsoft),
+  [Laurent Quérel](https://github.com/lquerel) (F5)
+draft: true
```

Comment:
It's best to add these names to the frontmatter of the blog post and remove the entries from repo-wide `.cspell` file. We try to reserve the `.cspell` file for words that are common across our documentation and blogs.

```suggestion
cSpell:ignore: Greptime OTAP Quérel
draft: true
```

Reply:
Sorry, I get it now. Thank you.